daemon/daemon_main.go

```diff
@@ -901,16 +916,61 @@ func initEnv(cmd *cobra.Command) {
 		log.WithError(err).Fatal("Invalid sidecar-istio-proxy-image regular expression")
 		return
 	}
+
+	if option.Config.PolicyEnforcementInterface == "" {
+		option.Config.PolicyEnforcementInterface = viper.GetString("policy-enforcement-interface")
```

Comment:
Was this rebased against the config changes? Don't we configure these under `option.Config.Populate()` these days?